src/plugins/explorer/sync-transactions.js

```diff
@@ -109,23 +146,11 @@ function createSyncer (config, eventBus, web3, queue, eventsRegistry, indexer) {
             debug(`Could not get past events for ${eventName}`)
             return null
           }
-
           return () =>
-            contract
-              .getPastEvents(eventName, {
-                fromBlock: Math.max(fromBlock, minBlock),
-                toBlock: Math.max(toBlock, minBlock),
-                filter
-              })
-              .then(function (events) {
-                debug(`${events.length} past ${eventName} events retrieved`)
-                return Promise.all(
-                  events.map(queue.addEvent(address, metaParser))
-                )
-              })
+            getPastEventsWithChunks({ address, contract, eventName, fromBlock, toBlock, filter, minBlock, onProgress, metaParser })
```

Comment:
Check the code format here. Didn't the linter/formatter catch this?

Reply:
I just checked, it throws a warning (eslint) but the automatic fix does not fix it. Prettier is not installed though

Reply:
Fixed here https://github.com/autonomoussoftware/metronome-wallet-core/pull/32/commits/d210679451505871c1f651ee8771624fe4ceb25d